addons/mail/static/src/core/common/message_model.js

```diff
@@ -43,6 +43,11 @@ export class Message extends Record {
     }
     body = fields.Html("");
     call_history_ids = fields.Many("discuss.call.history");
+    channel_id = fields.One("discuss.channel", {
```

Comment:
This should be in discuss folder. And ideally all parts of the code making use of it should be in discuss folder too (but they can be moved at a later time).